example/app/controllers/Login.scala

```diff
@@ -31,11 +32,14 @@ class Login @Inject() (override val wsClient: WSClient, override val conf: Confi
    * (see `Application.scala`).
    */
   def oauth2Callback = Action.async { implicit request =>
-    // processOauth2Callback()  // without Google group membership checks
-    processOauth2Callback(requiredGoogleGroups, groupChecker)  // with optional Google group checks
+     processOauth2Callback()  // without Google group membership checks
```

Comment:
No, just laziness and the fact that the existing implementation in the example app was a bit ropey.